qt/scientific_interfaces/ISISReflectometry/test/Common/DecoderTest.h

```diff
@@ -41,9 +41,10 @@ class DecoderTest : public CxxTest::TestSuite {
   static DecoderTest *createSuite() { return new DecoderTest(); }
   static void destroySuite(DecoderTest *suite) { delete suite; }
 
-  DecoderTest() {
-    PyRun_SimpleString("import mantid.api as api\n"
-                       "api.FrameworkManager.Instance()");
+  void setUp() override { Mantid::API::AlgorithmFactory::Instance().subscribe<ReflectometryISISLoadAndProcess>(); }
```

Comment:
Thanks for these changes. I think this is actually better for the tests as they don't rely so much on other complicated external entities.